identity-service/src/notifications/components/notifications/Notification.jsx

```diff
@@ -148,10 +172,88 @@ const getMessage = (notification) => {
   return getNotificationMessage(notification)
 }
 
+const getTitle = (notification) => {
+  switch (notification.type) {
+    case NotificationType.RemixCreate: {
+      const { parentTrack } = notification
+      return (
+        <span className={'notificationText'}>
+          <BodyText text={`New remix of your track `} />
+          <HighlightText text={parentTrack.title} />
+        </span>
+      )
+    }
+    default: 
+      return null
+  }
+}
+
+const getTrackMessage = (notification) => {
+  switch (notification.type) {
+    case NotificationType.RemixCosign: {
+      const { remixTrack } = notification
+      return (
+        <span className={'notificationText'}>
+          <HighlightText text={remixTrack.title} />
+        </span>
+      )
+    }
+    default: 
+      return null
+  }
+}
+
+export const getTrackLink = (track) => {
+  return `https://audius.co/${track.route_id}`
```

Comment:
route_id doesn't have the track id appended to the end, so this won't work, will it?